src/styles/components/_toc.scss

```diff
@@ -40,6 +34,31 @@ main.w-toc-open {
 web-table-of-contents {
   display: none;
 
+  h2 {
+    line-height: 32px;
+  }
+
+  .w-scroll-spy__active {
+    position: relative;
+    -webkit-margin-before: 0em;
+    -webkit-margin-after: 0em;
+    -webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
```

Comment:
are these vendor prefixed logical properties? https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties

Reply:
No, this is rather to control the border height on the list-item element.

Reply:
I'm a bit confused... when I deleted them it didn't seem to change anything.
The reason I'm bringing it up is because I've tended to avoid using vendor prefixed properties if I can help it and it seems like these wouldn't work in Firefox. But I feel like I'm missing something.. can you show an example of what they do on the site?